.github/workflows/update-codeflare-operator.yml

```diff
@@ -0,0 +1,59 @@
+name: Update CodeFlare operator with latest changes from MCAD
+
+on:
+  workflow_run:
+    workflows: [Build and Push dev MCAD image into Quay]
+    types:
+      - completed
+
+jobs:
+  update-codeflare-operator:
```

Comment:
Should it be done only for workflow runs that target the main branch?

Long term, we'll may need to generalise it so it works with our branching / versioning strategy.

Reply:
Added a condition making sure that `Build and Push dev MCAD image into Quay` was triggered on `main` branch.